src/command/archive/push/push.c

```diff
@@ -390,7 +390,7 @@ cmdArchivePush(void)
             {
                 THROW_FMT(
                     ArchiveTimeoutError, "unable to push WAL file '%s' to the archive asynchronously after %s second(s)",
-                    strZ(archiveFile), strZ(strNewDbl((double)cfgOptionInt64(cfgOptArchiveTimeout) / MSEC_PER_SEC)));
+                    strZ(archiveFile), strZ(cfgOptionDisplay(cfgOptArchiveTimeout)));
```

Comment:
There is an inconsistency with archive-timeout and it is not in this file but in the archive/common.c. When I set the archive-time=0.1 in the pgbackrest.conf file, then run the check command I get:
P00  ERROR: [082]: WAL segment 000000020000000000000005 was not archived before the **100ms** timeout

But if I run `pgbackrest help archive-push archive-timeout` it states **current: 0.1** 
And if I change the archivePushTest, line 698 to `strLstAddZ(argList, "--archive-timeout=0.1");` I get:
ArchiveTimeoutError: unable to push WAL file 'bogus' to the archive asynchronously after 0.1 second(s)

Since I entered 0.1, I expected I would always see 0.1 second(s) as a display - not 100ms sometimes and 0.1 second(s) other times.

Reply:
That's true, but that mismatch was not introduced by this commit. I didn't want to change any messages (except load.c which refers to options) in this commit because it then it is harder to evaluate the change.

Reply:
And this is a place where cfgOptionDisplayVar() could be used. The difference between these two places is in one the code is aware that it is working with an option, and in the other place it is not. Again, that hasn't changed for this commit but it would be good to have a general purpose solution.

Reply:
If it is not appropriate for this commit, then when will it get done? The goal was to display the exact value displayed by the user.

Reply:
As discussed, added a card to make these more consistent. https://github.com/pgbackrest/pgbackrest/projects/2#card-59949268